lib/pbench/server/api/resources/users_api.py

```diff
@@ -469,20 +471,25 @@ def delete(self, username):
             abort(403, message="Not authorized to delete user")
 
         try:
-            user = User.query(username=username)
-            # Do not delete if the user is admin
-            if not user.is_admin():
-                User.delete(username)
```

Comment:
I still think that this is the correct place to implement the deletion _policy_, not in `User.delete()`.